src/core/thread/mesh_forwarder.cpp

```diff
@@ -396,15 +396,6 @@ otError MeshForwarder::UpdateIp6Route(Message &aMessage)
     return error;
 }
 
-void MeshForwarder::SetRxOff(void)
-{
-    ThreadNetif &netif = GetNetif();
-
-    netif.GetMac().SetRxOnWhenIdle(false);
-    mDataPollManager.StopPolling();
-    netif.GetSupervisionListener().Stop();
-}
```

Comment:
Are we sure we do not need to call the `StopPolling()` and `SupervisionLisner().Stop()`?

Reply:
Checking the source code, it seems to be ok.

We do this already from `MeshForwarder::SetRxOnWhenIdle()`